ruleset/sca/amazon/cis_amazon_linux_2.yml

```diff
@@ -928,256 +897,162 @@ checks:
       - More detailed documentation on OpenLDAP is available at https://www.openldap.org
     condition: none
     rules:
-      - 'c:systemctl is-enabled slapd ->  r:enabled'
+      - 'p:slapd'
 
 
 
-# 2.1.7 Ensure nfs-utils is not installed or the nfs-server service is masked (Automated)
-  - id: 20551 
+# 2.1.17 Ensure nfs-utils is not installed or the nfs-server service is masked (Automated)
+  - id: 20551
     title: "Ensure NFS and RPC are not enabled"
     description: "The Network File System (NFS) is one of the first and most widely distributed file systems in the UNIX environment. It provides the ability for systems to mount file systems of other servers through the network."
     rationale: "If the system does not export NFS shares or act as an NFS client, it is recommended that these services be disabled to reduce remote attack surface."
     remediation: "Run the following commands to disable nfs, nfs-serverand rpcbind: # systemctl disable nfs # systemctl disable nfs-server # systemctl disable rpcbind"
     compliance:
-      - cis: ["2.1.7"]
+      - cis: ["2.1.17"]
       - cis_csc: ["9.2"]
       - pci_dss: ["2.2.2"]
       - nist_800_53: ["CM.1"]
       - tsc: ["CC5.2"]
     condition: none
     rules:
-      - 'c:ystemctl is-enabled nfs ->  r:enabled'
-      - 'c:ystemctl is-enabled nfs-server ->  r:enabled'
-      - 'c:ystemctl is-enabled rpcbind ->  r:enabled'
+      - 'p:nfs'
+      - 'p:nfs-server'
+      - 'p:rpcbind'
 
-# 2.1.8 Ensure DNS Server is not enabled (Automate)
-  - id: 20552 
+# 2.1.7 Ensure DNS Server is not enabled (Automate)
```

Comment:
Checks from 2.1.7 to 2.1.13 of the CIS Benchmark could be located after 20550 to follow the logical order, avoiding having the check 2.1.17 (of the benchmark) between them.

We could also add the checks 2.1.14 and 2.1.15 of the CIS benchmark to complete the family.